pkg/executor/executor_shared.go

```diff
@@ -51,3 +51,14 @@ func checkIfProcessFailedToExecute(command string, executorName string, handle T
 
 	return nil
 }
+
+// getWaitTimeoutChan returns channel for timeout in Wait(timeout) function in TaskHandles.
+// When timeout is 0, then timeout will never occur.
+func getWaitTimeoutChan(timeout time.Duration) <-chan time.Time {
```

Comment:
@skonefal rename to `createTimeoutChan()`, please (the function itself has nothing to do with waiting).